# HG changeset patch
# User Sergey Mironov <grrwlf@gmail.com>
# Date 1383731647 -14400
#      Wed Nov 06 13:54:07 2013 +0400
# Node ID 2dd2aaedef8921b4091b76aa50e67db066eb10f1
# Parent  01b5d3aebdda6747c2a41580bfa822f7ea006b5c
Add -print-cinlude command line option

diff --git a/src/main.mlton.sml b/src/main.mlton.sml
--- a/src/main.mlton.sml
+++ b/src/main.mlton.sml
@@ -58,6 +58,8 @@
 			              raise Code OS.Process.success)
         fun printCCompiler () = (print ((Settings.getCCompiler ()) ^ "\n");
 			              raise Code OS.Process.success)
+        fun printCInclude () = (print ((Config.includ) ^ "\n");
+			              raise Code OS.Process.success)
 
         fun doArgs args =
             case args of
@@ -71,6 +73,8 @@
                  doArgs rest)
               | "-print-ccompiler" :: rest =>
                   printCCompiler ()
+              | "-print-cinclude" :: rest =>
+                  printCInclude ()
               | "-ccompiler" :: ccomp :: rest =>
                 (Settings.setCCompiler ccomp;
                  doArgs rest)
